ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/HeartbeatAgent.java

```diff
@@ -223,6 +223,9 @@ private void processHeartbeats(final long windowStart, final long windowEnd) {
           }
           return status;
         });
+        for (HostStatusListener listener : hostStatusListeners) {
+          listener.onHostStatusUpdated(getHostsStatus());
```

Comment:
any exception handling needed here to ensure we don't stop notifying if one of  the callbacks throw an error?

Reply:
Done.  Factored out notification to a method where I now catch and log any errors.